page_playlist.go

```diff
@@ -248,6 +248,11 @@ func (p *PlaylistPage) UpdatePlaylists() {
 		if err != nil {
 			p.logger.PrintError("GetPlaylists", err)
 		}
+		if response == nil {
```

Comment:
This may be gonic related, but very rarely I actually hit this: the query does not return an error, but it also returns nil response object. It is almost certainly a bug in the bespoke OpenSubsonic library which should be tracked down, but even so, this prevents panics.